Makefile

```diff
@@ -37,6 +37,16 @@ FFI_DEPS:=$(addprefix $(FFI_PATH),$(FFI_DEPS))
 
 $(FFI_DEPS): build/.filecoin-install ;
 
+ifeq ($(shell uname -s), Darwin)
+export CGO_ENABLED=1
+    ifeq ($(shell uname -m), arm64)
+export GOARCH=arm64
+export LIBRARY_PATH=$(shell brew --prefix)/lib
+export FFI_BUILD_FROM_SOURCE=1
```

Comment:
The mac libraries actually support arm64, and at least for me they work just fine, there shouldn't be a need to build from source on M1/M2 macs

```
$ lipo -info extern/filecoin-ffi/libfilcrypto.a
Architectures in the fat file: extern/filecoin-ffi/libfilcrypto.a are: x86_64 arm64
```

Reply:
@kasteph could you remove this line and re-test?